docs/workspaces.md

```diff
@@ -15,76 +35,97 @@ coder create --template="<templateName>" <workspaceName>
 
-By default, workspaces are manually turned on/off by the user. However, a schedule
-can be defined on a per-workspace basis to automate the workspace start/stop.
+To set a workspace's schedule, go to the workspace, then **Settings** > **Schedule**.
 
 ![Scheduling UI](./images/schedule.png)
 
-### Autostart
+Coder might also stop a workspace automatically if there is a [template
+update](./templates/index.md#Start/stop) available.
+
+
+### Autostart and autostop
 
-The autostart feature automates the workspace build at a user-specified time
-and day(s) of the week. In addition, users can select their preferred timezone.
+Use autostart to start a workspace at a specified time and which days of the
+week. Also, you can choose your preferred timezone.
 
 ![Autostart UI](./images/autostart.png)
 
-### Autostop
+Use autostop to stop a workspace after a number of hours. Autostop
+won't stop a workspace if you're still using it. It waits for
+another hour before checking again. Coder checks for active
+connections in the IDE, SSH, Port Forwarding, and coder_app.
 
-The autostop feature shuts off workspaces after given number of hours in the "on"
-state. If Coder detects workspace connection activity, the autostop timer is bumped up
-one hour. IDE, SSH, Port Forwarding, and coder_app activity trigger this bump.
+![Autostop UI](./images/autostop.png)
 
-![autostop UI](./images/autostop.png)
 
 ### Max lifetime
 
-Max lifetime is a template-level setting that determines the number of hours a
-workspace can run before it is automatically shutdown, regardless of any
-active connections. This setting ensures workspaces do not run in perpetuity
-when connections are left open inadvertently.
+Max lifetime is a template setting that determines the number of hours
+a workspace will run before Coder automatically stops it, regardless
+of any active connections. Use this setting to ensure that workspaces
+do not run in perpetuity when connections are left open inadvertently.
+
 
 ## Updating workspaces
 
-Use the following command to update a workspace to the latest template version.
-The workspace will be stopped and started:
+After updating the default version of the template that a workspace
+was created from, you can update the workspace.
+
+![Updating a workspace](./images/workspace-update.png)
+
+If the workspace is running, Coder stops it, updates it, then starts
+the workspace again.
+
+On the command line:
 
 ```console
 coder update <workspace-name>
 ```
 
+
 ## Repairing workspaces
 
-Use the following command to re-enter template input
-variables in an existing workspace. This command is useful when a workspace fails
-to build because its state is out of sync with the template.
+Use the following command to re-enter template input variables in an
+existing workspace. This command is useful when a workspace fails to
+build because its state is out of sync with the template.
 
 ```console
 coder update <your workspace name> --always-prompt
 ```
```

Comment:
Perhaps we should replace this section with the UI, you can edit these things in "Template Settings"

Reply:
@bpmct Good idea.

Please confirm: are we talking about re-entering [parameters](https://coder.com/docs/v2/latest/templates/parameters) or [template variables](https://coder.com/docs/v2/latest/templates/parameters#terraform-template-wide-variables)?

Reply:
This is template variables

Reply:
Thank you. I've added a screenshot and updated the text.